collector/lib/CollectorService.cpp

```diff
@@ -73,6 +76,9 @@ void CollectorService::RunForever() {
       return;
     }
     CLOG(INFO) << "Sensor connectivity is successful";
+
+    runtime_control_service.Init(config_.grpc_channel);
```

Comment:
Just leaving a thought: The configuration when it is received from Sensor could be exchanged in an initial handshake between Sensor<>Collector.
(Is there an initial handshake between collector/sensor? :thinking: )